presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,436 @@ Except for the limit on queued queries, when a resource group runs out of a reso
-    * ``INSERT``: ``INSERT`` and ``CREATE TABLE AS`` queries.
-    * ``SELECT``: ``SELECT`` queries.
+* ``queryType`` (optional): String to match against the type of the query submitted:
 
-* ``clientTags`` (optional): list of tags. To match, every tag in this list must be in the list of
+  - ``SELECT``: ``SELECT`` queries.
+  - ``EXPLAIN``: ``EXPLAIN`` queries (but not ``EXPLAIN ANALYZE``).
+  - ``DESCRIBE``: ``DESCRIBE``, ``DESCRIBE INPUT``, ``DESCRIBE OUTPUT``, and ``SHOW`` queries.
+  - ``INSERT``: ``INSERT``, ``CREATE TABLE AS``, and ``REFRESH MATERIALIZED VIEW`` queries.
+  - ``UPDATE``: ``UPDATE`` queries.
+  - ``DELETE``: ``DELETE`` queries.
+  - ``ANALYZE``: ``ANALYZE`` queries.
+  - ``DATA_DEFINITION``: Queries that alter/create/drop the metadata of schemas/tables/views,
+    and that manage prepared statements, privileges, sessions, and transactions.
+
+* ``clientTags`` (optional): List of tags. To match, every tag in this list must be in the list of
   client-provided tags associated with the query.
 
-* ``group`` (required): the group these queries will run in.
+
+Selectors are processed sequentially and the first one that matches will be used.
 
 Global Properties
 -----------------
 
 * ``cpuQuotaPeriod`` (optional): the period in which cpu quotas are enforced.
+  The ``cpuQuotaPeriod`` is a global property often used in container-based environments to control the amount
+  of CPU resources that a container can use in a specified period.
 
-Selectors are processed sequentially and the first one that matches will be used.
+Please note that the exact implementation and naming of these properties can
+vary between different container runtimes and orchestration systems.
 
 Providing Selector Properties
 -----------------------------
 
 The source name can be set as follows:
 
-  * CLI: use the ``--source`` option.
+* CLI: use the ``--source`` option.
+
+* JDBC driver when used in client apps: add the ``source`` property to the
+  connection configuration and set the value when using a Java application that
+  uses the JDBC Driver.
 
-  * JDBC: set the ``ApplicationName`` client info property on the ``Connection`` instance.
+* JDBC driver used with Java programs: add a property with the key ``source``
+  and the value on the ``Connection`` instance as shown in :ref:`the example
+  <jdbc-java-connection>`.
 
 Client tags can be set as follows:
 
-  * CLI: use the ``--client-tags`` option.
+* CLI: use the ``--client-tags`` option.
 
-  * JDBC: set the ``ClientTags`` client info property on the ``Connection`` instance.
+* JDBC driver when used in client apps: add the ``clientTags`` property to the
+  connection configuration and set the value when using a Java application that
+  uses the JDBC Driver.
+
+* JDBC driver used with Java programs: add a property with the key
+  ``clientTags`` and the value on the ``Connection`` instance as shown in
+  :ref:`the example <jdbc-parameter-reference>`.
 
 Example
--------
+^^^^^^^
```

Comment:
nit: could this be `---` and the following `File Resource Group Manager` and `Database Resource Group Manager` change to `^^^` ?